syntax = "proto3";

package upper;

option go_package = "github.com/griddic/grpc_playground";


service Upper {
  rpc ListThousand (ListRequests) returns (Numbers) {};
};

message ListRequests {
  int64 page_size = 2;
  string page_token = 3;
  string filters = 4;
}
message Numbers {
  repeated int64 nums = 1;
  string next_page_token = 2;
}
